alteryx/compose
# v0.7.0

**v0.7.0** November 2, 2021
  * Enhancements
    * Add ``maximum_data`` parameter to control when a search should stop (#216)
    * Add optional automatic update checker (#223, #229, #232)
    * Varying first cutoff time for each target group (#258)
  * Documentation Changes
    * Update doc tutorials to the latest API changes (#227)
    * Pin documentation requirements to avoid warnings and breaking changes (#244)
  * Testing Changes
    * Check if release notes were updated (#217)
    * Add minimum dependency checker to generate minimum requirement files (#218)
    * Add CI workflow for unit tests with minimum dependencies (#220)
    * Create separate workflows for each CI job (#220)
    * Pass token to authorize uploading of codecov reports (#226)
    * Update minimum unit tests to run on all pull requests (#230)
    * Add workflow to check latest dependencies (#233)
    * Update reviewers for minimum and latest dependency checkers (#257)

Thanks to the following people for contributing to this release:
@gsheni, @jeff-hernandez
